lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -372,6 +393,47 @@ putTx inputs outputs = do
         putAddress address
         putWord64be $ getCoin coin
 
+{-------------------------------------------------------------------------------
+                                 Stake Distribution
+-------------------------------------------------------------------------------}
+
+data StakeDistribution = StakeDistribution
+    { epoch :: Word64
+    , stake :: StakePools
+    } deriving (Show, Eq, Generic)
+
+-- | Coins are stored as Lovelace (reminder: 1 Lovelace = 1e-6 ADA)
+newtype Stake = Stake { getStake :: Word64 } deriving (Show, Eq)
+
+data StakePools = StakePools
```

Comment:
On very first glance, this name seems a bit weird? Isn't this more like `StakeDistribution` since you're talking about `dangling`/`pools`/`unassigned` - which are all things that have stake?

Keeping the `epoch` separate from the distribution, as you've done, sounds like it could be a good idea, but I'm not exactly sure why.

Reply:
you are right, reconsidered it. and introdced `StakeApiResponse` and renamed `StakePools` into `StakeDistribution`